File: electronics.rest-api/src/main/java/edu/sdr/electronics/dto/response/QuizWrapperResponse.java
package edu.sdr.electronics.dto.response;

import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class QuizWrapperResponse {

    private List<QuizQuestionResponse> questions;
    private QuizCompletionResponse quizCompletion;
}

File: electronics.ui/src/app/quiz/quiz-wrapper-response.ts
import {QuizQuestion} from "./quiz-question-response.model";
import {QuizCompletionResponse} from "./quiz-completion.response";

export class QuizWrapper {
  questions: Array<QuizQuestion> = [];
  quizCompletion: QuizCompletionResponse = new QuizCompletionResponse();
}
